Contants/Texts/Source/texts/Skills_PostAction.txt

```diff
@@ -56,4 +56,9 @@ the character to the target[X]
 ## MSG_SKILL_Lifetaker
 Lifetaker:[NL]
 Restore up to 25% current HP after[NL]
-attacking and defeating an enemy.[X]
\ No newline at end of file
+attacking and defeating an enemy.[X]
+
+## MSG_SKILL_ReMove
+Re-Move:[NL]
+Gain an additional action at the end[NL]
+of this unit's turn (Skill % activation).[X]
```

Comment:
a Blank line is required at the end of the file.

Reply:
Done